### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by MagicLike**
- [x] Formatting updates
- [x] Renamed docs folder
- [x] Reworded bootloader documentation
- [x] Add own bootloader policy docs
- [x] Use GitHub's highlighted blockquotes
- [x] Add vendor lock disclamer from website

[Preview of the highlighted blockquotes](https://gist.github.com/MagicLike/c3f5ee530e5e3efc4ee5efdb5606db43)

### CODE REVIEW

File: docs/unlocking_the_bootloader.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| OnePlus | Fastboot | OEM/Flashing unlock | No | All |
+| Fairphone | Fastboot | [Code from manufacturer](https://support.fairphone.com/hc/en-us/articles/10492476238865-Manage-the-Bootloader)<br>(**Already unlocked until FP2**) | No | All |
+| Xiaomi | Fastboot | [Mi Unlock Tool](https://new.c.mi.com/global/post/101245) | **Yes** | Mi 4c, Redmi Note 3, Mi Note Pro, Redmi 3, Mi 4S, Mi 5 and all devices from 2016 onwards |
+| Motorola | Fastboot | [Code from manufacturer](https://en-us.support.motorola.com/app/standalone/bootloader/unlock-your-device-a) | **Yes** | Almost all **except** carrier specific models (e.g. Verizon, AT&T, Tracfone) and certain other models |
+| Sony | Fastboot | [Code from manufacturer](https://developer.sony.com/open-source/aosp-on-xperia-open-devices/get-started/unlock-bootloader) | **Yes** | All **except** XQ-CT62 (1Ⅳ US) & XQ-CQ62 (5Ⅳ US) |
+| Huawei | Fastboot | No official codes since 2017/2018<br>(**Unofficial methods available**) | **Yes** | Mate 9/9Pro, P10/P10Plus |
+| Honor | Fastboot | No official codes since 2017/2018<br>(**Unofficial methods available**) | **Yes** | Honor 8 |
+| LG | --- | Impossible since December 2021 | **Yes** |  |
```
Comment (tsterbak) [inline docs/unlocking_the_bootloader.md:24]:
```suggestion
| LG | Fastboot | Impossible since December 2021 | **Yes** |  |
```

Comment (tsterbak) [reply docs/unlocking_the_bootloader.md:24]: At least I flashed my LG phone with fastboot :)

File: docs/unlocking_the_bootloader.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+For some devices, it is necessary to unlock the bootloader manually. You usually need to create an account on the manufacturer's website, wait for a certain time and obtain a code from there. OpenAndroidInstaller will try to guide you as far as possible. These vendors include *Sony, Motorola and Xiaomi* amongst others.
+
+Other phone manufacturers no longer allow you to unlock the bootloader. Nothing can be done if you have not unlocked your device in time. These manufacturers include *Huawei, Honor, LG and ASUS*. Support for these manufacturers will always be very limited.
+
+<details>
+    <summary>
+        <b>Here is a brief overview of the bootloader policies of some of the most popular brands (may be incomplete/inaccurate)</b>
+    </summary>
+
+| Brand | Flashing tool | Unlocking Method | Loss of guarantee\* | Supported models |
```
Comment (tsterbak) [inline docs/unlocking_the_bootloader.md:13]: What is this referring to? Support for unlocking or the installer?
Not sure if we want to maintain a list of devices that are bootloader unlockable or not.

File: docs/unlocking_the_bootloader.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+| Honor | Fastboot | No official codes since 2017/2018<br>(**Unofficial methods available**) | **Yes** | Honor 8 |
+| LG | --- | Impossible since December 2021 | **Yes** |  |
+| ASUS | Fastboot | Impossible since May 2023 (ASUS unlocking App) | **Yes** |  |
+
+**\*** In the EU you won't loose your standard 2 years of the warranty when you unlock your bootloader, flash your device or root it. ([source](https://forum.xda-developers.com/t/info-eu-rooting-and-flashing-dont-void-the-warranty.1998801/))
+
+</details>
+<br>
+
+You can find more information about the brands and their bootloader policies [here](https://wikilibriste.fr/fr/tutoriels-android/bootloader-unlock). (in French)
```
Comment (tsterbak) [inline docs/unlocking_the_bootloader.md:32]:
```suggestion
You can find more information about the brands and their bootloader policies [here](https://wikilibriste.fr/fr/tutoriels-android/bootloader-unlock) (in French).
```

**Comment by tsterbak** [comment]
Looks good, in general. I requested some small changes.

Also, if you change the paths of the documentations, the website should be updated as well.

File: docs/unlocking_the_bootloader.md (hunk 2 above)
Comment (MagicLike) [reply docs/unlocking_the_bootloader.md:13]: To what exactly are you referring to?

**Comment by MagicLike** [comment]
> Also, if you change the paths of the documentations, the website should be updated as well.

Two questions regarding that: 
Should I also update the documentation text on the website?
Should I only edit in the `templates` folder? If so, how does a website build get triggered?

File: docs/unlocking_the_bootloader.md (hunk 2 above)
Comment (MagicLike) [reply docs/unlocking_the_bootloader.md:13]: Ohhh do you mean the `Supported models`?
This is just the data I took from the website linked below and combined it with my own and the internet's data... :D 
There is not really a need to actively maintain it, as not much will change in the future, as the listed brands either allow or don't allow unlocking (anymore)...

**Comment by tsterbak** [comment]
LGTM :) 
The only issue is that it is branched off of the `dev` branch so this cannot be released before the next release. Can you apply the changes to a branch based on `main`, then this can be merged quickly :)

**Comment by tsterbak** [comment]
> Should I also update the documentation text on the website? - I have done it already for convenience :D
Should I only edit in the templates folder? If so, how does a website build get triggered?

Currently there is no online automated build. I edit the templates and then run `make website` to create and update the public files. then I commit both.

**Comment by MagicLike** [comment]
> > Should I also update the documentation text on the website? - I have done it already for convenience :D
> Should I only edit in the templates folder? If so, how does a website build get triggered?
> 
> Currently there is no online automated build. I edit the templates and then run `make website` to create and update the public files. then I commit both.

Shouldn't this be automated via GH-Actions?

**Comment by tsterbak** [comment]
> Shouldn't this be automated via GH-Actions?

Yeah, could be. But so far I basically always wanted to view the changes locally either way (and I think this is reasonable to do so). So I didn't need the build automation.

**Comment by MagicLike** [comment]
> LGTM :) 
> The only issue is that it is branched off of the `dev` branch so this cannot be released before the next release. Can you apply the changes to a branch based on `main`, then this can be merged quickly :)

I mean you can also just cherry pick my commits if this branch is used, but if not, I am also fine with waiting those few days until the 0.4.5 release :)

**Comment by MagicLike** [comment]
> > Shouldn't this be automated via GH-Actions?
> 
> Yeah, could be. But so far I basically always wanted to view the changes locally either way (and I think this is reasonable to do so). So I didn't need the build automation.
> 
> But the current workflow could at least be documented in the website readme... :grimacing: 

I'll put it on the overhaul list 😁

**Review by tsterbak — APPROVED** [review]

**Pull request merged**